phpunit.xml.dist

```diff
@@ -18,15 +19,16 @@
 
     <php>
         <var name="driver_config_factory" value="Mink\WebdriverClassDriver\Tests\WebdriverClassicConfig::getInstance"/>
-        <!--server name="WEB_FIXTURES_HOST" value="http://test.mink.dev" /-->
+
+        <server name="WEB_FIXTURES_HOST" value="http://host.docker.internal:8002"/>
         <!-- MacOS -->
         <!--<server name="WEB_FIXTURES_HOST" value="http://docker.for.mac.localhost:8002"/>-->
-        <!--<server name="WEB_FIXTURES_BROWSER" value="firefox"/>-->
+
         <!-- where driver will connect to -->
         <server name="DRIVER_URL" value="http://localhost:4444/wd/hub"/>
+
         <!-- where DocumentRoot of 'Test Machine' is mounted to on 'Driver Machine' (only if these are 2 different machines) -->
-        <!--server name="DRIVER_MACHINE_BASE_PATH" value="" /-->
-        <!--server name="TEST_MACHINE_BASE_PATH" value="" /-->
+        <server name="DRIVER_MACHINE_BASE_PATH" value="/fixtures/"/>
     </php>
```

Comment:
Defining ENV vars with Docker defaults prevents a developer from running a test suite with a locally started (not Docker-based) Selenium server.

I recommend replacing with code with the code of MinkSelenium2Driver.